specification/trace/semantic_conventions/messaging.md

```diff
@@ -80,6 +80,48 @@ Often such destinations are unnamed or have an auto-generated name.
 
 Given these definitions, the remainder of this section describes the semantic conventions for Spans describing interactions with messaging systems.
 
+### Context propagation
+
+A message may pass many different components and layers in one or more intermediaries
+when it is propagated from the producer to the consumer(s). To be able to correlate
+consumer traces with producer traces using the existing context propagation mechanisms,
+all components must propagate context down the chain.
+
+This imposes a difficulty because it cannot be assumed, and in many cases,
+it is not even desired, that all components and layers to be instrumented and
+propagate context according to OpenTelemetry requirements.
+
+To be able to correlate consumer traces with producer traces without requiring
+intermediary instrumentation, the context needs to be propagated on a
+*per-message* basis instead of on a *per-request* basis. This allows all components
+to have access to the same per-message context information, making it possible
+to correlate all the stages involved in processing a message with the message's
+creation.
+
+A message *creation context* allows correlating producer with consumer(s)
+of a message and model the dependencies between them,
+regardless of the underlying messaging transport mechanism and its instrumentation.
+
+The message creation context is created by the producer and should be propagated
+to the consumer(s). Producer and consumer applications should be instrumented
+in a way so that the creation context is attached to messages and extracted
+from messages in a coordinated way.
+
+If the message creation context cannot be attached to the message and propagated,
+consumer traces cannot be directly correlated to producer traces.
+
+A producer SHOULD attach a message creation context to each message.
+The message creation context SHOULD be attached in a way so that it is
+not possible to be changed by intermediaries.
+
+> This document does not specify the exact mechanisms on how the creation context
+is attached/extracted to/from messages. Future versions of these conventions
```

Comment:
I am not sure what the standard is on OTEL, do you prefix every line in a note with `>`? It doesn't change the rendered version, I think. Did you run `markdownlint`?

Reply:
Good question. I always use `>` on the first line. Markdown lint didn't complain as well.

Reply:
I looked, it seems it's every line is prefixed in the examples I found. Changed in [4d89927](https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/22/commits/4d89927be29348fe8a06aee2193b170a3b6e7fc1)